.prettierrc.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
what would be cool would be if this was stored someplace like satchel and could be shared via climbers to other microsites

Reply:
Agreed, we could have an REI global one and everyone could opt in:
https://prettier.io/docs/en/configuration.html#sharing-configurations